tap_mailshake/client.py

```diff
@@ -64,6 +68,7 @@ class MailshakeInternalServiceError(MailshakeError):
     405: MailshakeMethodNotAllowedError,
     409: MailshakeConflictError,
     422: MailshakeUnprocessableEntityError,
+    'limit_reached': MailshakeAPILimitReachedError,
```

Comment:
Its a little weird that this is now a map of mixed keys.

Reply:
More accurately mapped the Mailshake errors and switched to exclusively using their error codes rather than statuses.